gradle/libs.versions.toml

```diff
@@ -36,9 +37,11 @@ swagger-annotations-jakarta = "2.2.15"
 titanium = "1.3.2"
 kafkaClients = "3.5.1"
 testcontainers = "1.18.3"
+tink = "1.10.0"
 
 
 [libraries]
+apicatalog-iron-vc = { module = "com.apicatalog:iron-verifiable-credentials", version.ref = "iron-vc" }
```

Comment:
do the `apicatalog` part needs to be used as the name? we didn't had that in the `titaniumJsonLd` library. `iron-vc` could be enough